server/api/user_handler.py

```diff
@@ -0,0 +1,56 @@
+from flask import jsonify, Blueprint, request
+from api import db, bcrypt
+from api.models import User
+
+user_handler = Blueprint("user_handler", __name__)
+
+
+@user_handler.route('/api/register', methods=['POST'])
+def register():
+    username = request.form.get("username")
+    password = request.form.get("password")
+    email = request.form.get("email")
+
+    # input missing
+    if username is None or password is None or email is None:
+        return jsonify({"Error": "required input missing"}), 400
+
+    # check input errors
+    if len(password) < 6:
+        return jsonify({"Error": "Password must have at least 6 characters."}), 400
+
+    if User.query.filter_by(email=email).first() is not None:
+        return jsonify({"error": "email exist"}), 400
+
+    if User.query.filter_by(username=username).first() is not None:
```

Comment:
I think you can utilize the power of SQL by merging the check of email and username in one go:

https://stackoverflow.com/a/7942571/14204824